packages/gnarly-bin/src/index.ts

```diff
@@ -55,6 +61,14 @@ const main = async () => {
     },
   })
 
+  const erc20Reducer = makeERC20Reducer(Keys.ZRX, makeERC20TypeStore(
+    Sequelize,
+    sequelize,
+  ))(
+    ZRX_ADDRESS,
```

Comment:
I hope using 0x token in bin is not an issue, I used that one because it has alot of traffic and its nice for testing